### DISCUSSION TITLE
Disk image

### DISCUSSION THREAD

**Discussion by Liujiaqi-jlu** [gem5/gem5#1773] [Q&A] [answered]
Hello, when I am building the image in full-system mode, it keeps prompting "qemu.initialize: Waiting for SSH to become available...". It might be due to poor network conditions. I would like to ask if there are any pre-built latest images available for download. Building it myself is really not ideal.Thanks！

**Comment by robhau** [comment, marked as answer]
Hi @Liujiaqi-jlu,
I suggest you to visit https://resources.gem5.org/. There you can find disk-images for different architectures.

1. Click on "View All" under "Getting Started".
2. Select "disk-image" as category
3. Select your ISA
4. Select your gem5 version

**Reply by Liujiaqi-jlu** [reply]
Thanks，I find it！
